packages/driver/src/multi-domain/cypress.ts

```diff
@@ -81,6 +84,16 @@ const onBeforeAppWindowLoad = (Cypress: Cypress.Cypress, cy: $Cy) => (autWindow:
   Cypress.state('window', autWindow)
   Cypress.state('document', autWindow.document)
 
+  // Can we assume this is always true within the switchToDomain command? I'm not sure thats always true. Should we always defer to the primary domain for this value?
+  // This is needed to be able to continue through the 'stabilityChanged' function to log the loading state for the page.
+  cy.state('duringUserTestExecution', true)
```

Comment:
I don't think we should _not_ assume it's always true, since the page could navigate between tests. 

This and `redirectionCount` below should probably be inherited from the primary domain. I think for the time being, let's leave these values as-is, but make sure to revisit once there are more example of them. Hopefully then we'll be in a better place to judge how exactly we should handle them.